### ISSUE TITLE
Epic: Extension integration and configuration options

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#9369]
Following up on the work already done on Extensions, a few items are missing in order to improve the "coverage" of extensions configuration.

- [x] https://github.com/rancher/dashboard/issues/9318
- [ ] https://github.com/rancher/dashboard/issues/9286
- [x] https://github.com/rancher/dashboard/issues/9290
- [x] https://github.com/rancher/dashboard/issues/8314
- [ ] https://github.com/rancher/dashboard/issues/9367
- [ ] https://github.com/rancher/dashboard/issues/9368 (only after github private repo is done...)
- [ ] https://github.com/rancher/dashboard/issues/9362
